flatdata-rs/lib/src/multiarrayview.rs

```diff
@@ -1,40 +1,42 @@
-use crate::{
-    arrayview::ArrayView,
-    structs::{IndexStruct, VariadicRef, VariadicRefFactory, VariadicStruct},
-};
+use crate::structs::{IndexStruct, VariadicRef, VariadicRefFactory, VariadicStruct};
 
-use std::{fmt, iter, marker, ops::RangeBounds};
+use std::{fmt, iter, marker};
 
 /// A read-only view on a multivector.
 ///
 /// For the detailed description of multivector and examples, cf.
 /// [`MultiVector`].
 ///
 /// [`MultiVector`]: struct.MultiVector.html
-#[derive(Clone)]
 pub struct MultiArrayView<'a, Ts>
 where
     Ts: VariadicRefFactory,
 {
-    index: ArrayView<'a, <Ts as VariadicStruct<'a>>::Index>,
+    index: &'a [Ts::Index],
     data: &'a [u8],
     _phantom: marker::PhantomData<Ts>,
```

Comment:
I think this phantom marker can be removed now.

Reply:
Done